addons/compat_gm/CfgVehicles.hpp

```diff
@@ -0,0 +1,478 @@
+        EGVAR(vehicle_damage,canHaveFireRing) = 0.1;
+    };  
+    class gm_tracked_Tank_base: gm_tracked_base {
+        EGVAR(vehicle_damage,hullDetonationProb) = 0.01;
+        EGVAR(vehicle_damage,turretDetonationProb) = 0.01;
+        EGVAR(vehicle_damage,engineDetonationProb) = 0.01;
+        EGVAR(vehicle_damage,hullFireProb) = 0.2;
+        EGVAR(vehicle_damage,turretFireProb) = 0.1;
+        EGVAR(vehicle_damage,engineFireProb) = 0.2;
+        EGVAR(vehicle_damage,detonationDuringFireProb) = 0.2;
+        EGVAR(vehicle_damage,canHaveFireRing) = 0.1;
+    };
+
+    // EAST
+    class gm_bmp1_base: gm_tracked_APC_base {
+        EGVAR(map,vehicleLightColor)[] = {0,1,0,0.1};
+        EGVAR(refuel,fuelCapacity) = 460;
+        EGVAR(cookoff,cookoffSelections)[] = {"mainturret_hatch","commanderturret_hatch"}; 
+    };
+    
+    class gm_pt76_base: gm_tracked_APC_base {
+        EGVAR(refuel,fuelCapacity) = 250;
+    };
+
+    class gm_t55_base: gm_tracked_Tank_base {
+        EGVAR(refuel,fuelCapacity) = 900; // only with external tanks
+        EGVAR(cookoff,cookoffSelections)[] = {"machinegunturret_01_hatch","commanderturret_hatch"}; 
+    };
+
+    class gm_zsu234_base: gm_tracked_Tank_base {
+        EGVAR(refuel,fuelCapacity) = 812;
+    };
+
+    // WEST
+    class gm_Leopard1_base;
+    class gm_Leopard1a0_base: gm_Leopard1_base {
+        EGVAR(refuel,fuelCapacity) = 955;
+        EGVAR(cookoff,cookoffSelections)[] = {"mainturret_hatch_1","commanderturret_hatch"}; 
+    };
+
+    class gm_Gepard_base: gm_Leopard1_base {
+        EGVAR(refuel,fuelCapacity) = 985;
+    };
+
+    class gm_BPz2_base;
+    class gm_BPz2a0_base: gm_BPz2_base {
+        EGVAR(refuel,fuelCapacity) = 1160;
+        EGVAR(cookoff,cookoffSelections)[] = {"commanderturret_hatch"}; 
+    };
+
+    class gm_marder1_base: gm_tracked_APC_base {
+        EGVAR(refuel,fuelCapacity) = 652;
+        EGVAR(cookoff,cookoffSelections)[] = {"observerturret_hatch", "commanderturret_hatch"};
+    };
+
+    class gm_m113_base: gm_tracked_APC_base {
+        EGVAR(refuel,fuelCapacity) = 360;
+    };
+
+    class gm_m113a1g_base;
+    class gm_m113a1g_medic_base: gm_m113a1g_base {
+        EGVAR(medical,medicClass) = 1;
+    };
+
+    class gm_m113a1dk_base;
+    class gm_m113a1dk_engineer_base: gm_m113a1dk_base {
+        transportRepair = 0;
+        EGVAR(repair,canRepair) = 1;
+    };
+
+    class gm_m113a1dk_medic_base: gm_m113a1dk_base {
+        EGVAR(medical,medicClass) = 1;
+    };
+
+    // H E L I C O P T E R S
+
+    class Helicopter_Base_F;
+    class Helicopter_Base_H: Helicopter_Base_F {
+        class EventHandlers;
+    };
+    class gm_helicopter_base: Helicopter_Base_H {};
+
+    // WEST
+
+    class gm_bo105_base: gm_helicopter_base {
+        EGVAR(map,vehicleLightColor)[] = {1,0,0,0.1};
+        EGVAR(refuel,fuelCapacity) = 3700;
+        EGVAR(fastroping,enabled) = 0;
+
+        class UserActions
+            {
```

Comment:
Braces on same line as class!

Reply:
fixed in a92e1ce0ee27d6114fa8ceafbbc33aad16f580a2